### PR TITLE
Fix/jaeger custom ns

### LINKED ISSUE #5935: Custom jaeger ns is not supporting in helm chart
document referred https://linkerd.io/2.10/tasks/automatically-rotating-webhook-tls-credentials/

```
helm upgrade --install linkerd-jaeger \
 linkerd-jaeger \
 --namespace linkerd-jaeger \
 --set installNamespace=false \
 --set webhook.externalSecret=true \
 --set-file webhook.caBundle=certs/webhook.crt \
 --version 2.10.0 
```

Release "linkerd-jaeger" does not exist. Installing it now.
Error: rendered manifests contain a resource that already exists. Unable to continue with install: Namespace "linkerd-jaeger" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "linkerd-jaeger"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "linkerd-jaeger"

### PR DESCRIPTION
**Pull request by jijeesh**
support jaeger custom namespace

Problem
I am installing Linkerd using helm chart by using this [document](https://linkerd.io/2.10/tasks/automatically-rotating-webhook-tls-credentials/)  with a custom cert-manager. so it shows an error like

helm upgrade --install linkerd-jaeger \
 linkerd-jaeger \
 --namespace linkerd-jaeger \
 --set installNamespace=false \
 --set webhook.externalSecret=true \
 --set-file webhook.caBundle=certs/webhook.crt \
 --version 2.10.0 

Release "linkerd-jaeger" does not exist. Installing it now.
Error: rendered manifests contain a resource that already exists. Unable to continue with install: Namespace "linkerd-jaeger" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "linkerd-jaeger"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "linkerd-jaeger"

Solution
 The installNamespace=false value is not declared in that chart's settings. so i enabled as like other helm chart
[PR link](https://github.com/linkerd/linkerd2/issues/5935)

Validation

image
image

Fixes #[GitHub issue ID] #5935

DCO Sign off

Signed-off-by: Jijeesh K A <jijeesh.ka@gmail.com>

```

git commit message: jaeger custom ns added

```

Fixes #5935

Signed-off-by: Jijeesh K A <jijeesh.ka@gmail.com>
```

### CODE REVIEW

**Review by adleong — COMMENTED** [review]
Thanks, @jijeesh!  Good catch!

A couple of small things:
* Since you updated the helm values you'll need to run `bin/helm-docs` to update the docs
* Please add a newline to the end of  `jaeger/charts/linkerd-jaeger/templates/namespace.yaml`
* You'll need to add the DCO signoff to you commits so that we can accept them

**Comment by jijeesh** [comment]
my merge is mess-up.. let me reset and do it again

File: jaeger/charts/linkerd-jaeger/templates/namespace.yaml [hunk 1]
```diff
@@ -7,3 +8,4 @@ metadata:
     linkerd.io/extension: jaeger
   annotations:
     linkerd.io/inject: enabled
+{{ end -}}
```
Comment (Pothulapati) [inline jaeger/charts/linkerd-jaeger/templates/namespace.yaml:11]: Can you add a new line at the end of file?

Comment (jijeesh) [reply jaeger/charts/linkerd-jaeger/templates/namespace.yaml]: done

**Review by Pothulapati — APPROVED** [review]
LGTM. Thanks @jijeesh for fixing it!

**Review by adleong — APPROVED** [review]
Thanks!

**Pull request merged**